components/shared/webrender/display_list.rs

```diff
@@ -17,6 +17,13 @@ pub enum ScrollSensitivity {
     ScriptAndInputEvents,
     /// This node can only be scrolled by script events.
     Script,
+    /// This node cannot be scrolled
+    None,
+}
+#[derive(Clone, Copy, Debug, Deserialize, PartialEq, Serialize)]
+pub struct AxesScrollSensitivity {
+    pub vertical: ScrollSensitivity,
+    pub horizontal: ScrollSensitivity,
```

Comment:
You're adding a new public data structure so it would be nice to add a simple docstring. How about this:

```rust
/// The [`ScrollSensitivity`] of particular node in the vertical and horizontal axes.
```